planning/behavior_path_planner_common/src/turn_signal_decider.cpp

```diff
@@ -669,14 +704,16 @@ std::pair<TurnSignalInfo, bool> TurnSignalDecider::getBehaviorTurnSignalInfo(
 
   // If shift length is shorter than the threshold, it does not need to turn on blinkers
   if (std::fabs(relative_shift_length) < p.turn_signal_shift_length_threshold) {
-    return std::make_pair(TurnSignalInfo{}, true);
+    return std::make_pair(
```

Comment:
Since `path.path.points.front()` and `path.path.points.back()` are called so often to make a pose, would you consider making a variable for them?

`const auto  first_path_pose =getPose(path.path.points.front())` and `const auto last_path_pose = getPose(path.path.points.back())`